spec/controllers/idv/in_person/address_controller_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+RSpec.describe Idv::InPerson::AddressController do
+  include InPersonHelper
+  let(:in_person_residential_address_controller_enabled) { true }
+  let(:pii_from_user) do
+    { first_name: '',
+      last_name: '',
+      dob: '',
+      identity_doc_address1: '',
+      identity_doc_address2: '',
+      identity_doc_city: '',
+      identity_doc_zipcode: '',
+      state_id_jurisdiction: '',
+      identity_doc_address_state: '',
+      state_id_number: '',
+      same_address_as_id: 'false' }
+  end
+  let(:user) { build(:user) }
+  let(:flow_session) do
+    { pii_from_user: pii_from_user }
+  end
+  let(:flow_path) { 'standard' }
+
+  before(:each) do
+    allow(IdentityConfig.store).to receive(:in_person_residential_address_controller_enabled).
+      and_return(true)
+    allow(subject).to receive(:current_user).
+      and_return(user)
+    allow(subject).to receive(:pii_from_user).and_return(pii_from_user)
+    allow(subject).to receive(:flow_session).and_return(flow_session)
+    allow(subject).to receive(:flow_path).and_return(flow_path)
+    stub_sign_in(user)
+    stub_analytics
+    allow(@analytics).to receive(:track_event)
+  end
+
+  describe 'before_actions' do
+    context '#render_404_if_not_in_person_residential_address_controller_enabled' do
```

Comment:
Nit- Maybe reword context- or delete it.  Not sure it is explaining anything additional the below context and it blocks are not already explaining.

Reply:
i thought the naming was awkward too so I'm glad you confirmed that feeling! i'll update it

Reply:
Update is much more readable.  Still a nit- not blocking- but what do you think about context '#render_404_if_in_person_residential_address_controller_enabled not set' rather than context '#render_404_if_in_person_residential_address_controller_enabled_not_set'? (removing snake case around not set)